pkg/api/server/v1alpha2/auth/rbac.go

```diff
@@ -38,22 +39,41 @@ import (
 // This checks RBAC permissions in the `results.tekton.dev` group, and assumes
 // checks are done at the namespace
 type RBAC struct {
-	authn authnclient.AuthenticationV1Interface
-	authz authzclient.AuthorizationV1Interface
+	impersonation bool
```

Comment:
Maybe call this `allowImpersonation` to avoid confusion.

Reply:
renamed to `allowImpersonation `